content/id/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Pengenalan Pod
+content_template: templates/concept
+weight: 10
+card: 
+  name: concepts
+  weight: 60
+---
+
+{{% capture overview %}}
+Halaman ini menyajikan ikhtisar dari `Pod`, objek terkecil yang dapat di *deploy* didalam objek model Kubernetes.
+{{% /capture %}}
+
+
+{{% capture body %}}
+## Memahami Pod
+
+Sebuah *Pod* adalah unit dasar di Kubernetes--unit terkecil dan paling sederhana didalam objek model Kubernetes yang dapat dibuat dan di *deploy*. Sebuah *Pod* merepresentasikan suatu proses yang berjalan didalam kluster.
+
+*Pod* membungkus sebuah kontainer (atau, di beberapa kasus, beberapa kontainer), sumber penyimpanan, alamat jaringan *IP* yang unik, dan opsi yang mengatur bagaimana kontainer harus dijalankan. *Pod* merupakan representasi dari unit *deployment*: *sebuah instance aplikasi didalam Kubernetes*, yang mungkin terdiri dari satu kontainer atau sekumpulan kontainer yang berbagi sumber daya.
```

Comment:
nit: *sebuah instance aplikasi didalam Kubernetes*

should be
```
sebuah *instance* aplikasi di dalam Kubernetes
```

Reply:
`sumber daya` distandarisasi menjadi *resource* sesuai glossary karena lebih mudah dimengerti